server/views/curfew/curfewAddressReview.pug

```diff
@@ -19,12 +19,16 @@ block content
     form(method="post")
       input(type="hidden" name="_csrf" value=csrfToken)
       input(type="hidden" name="bookingId" value=bookingId || '')
+      input(type="hidden" name="version" value=curfewAddressReviewVersion || '')
       div.pure-g.largePaddingBottom
         div.pure-u-1.pure-u-sm-5-12.pure-g
           div.form-group.inline.paddingTop
             if !licenceStatus.decisions.offenderIsMainOccupier
 
-              h3.heading-medium Does the main occupier consent to HDC?
+              if curfewAddressReviewVersion === '1'
+                h3.heading-medium Does the main occupier consent to HDC?
+              else 
+                h3.heading-medium Have you spoken to the main occupier and do they consent to HDC?
 
```

Comment:
Think it might be nicer to have 2 separate pug files - version 1/version 2 and reference the version in the routes file to work out which to show - like the risk management stuff - hopefully easier to see whats going on for future work as might be easy to miss this condition in one pug file - also easier to see from the file structure that there are 2 versions.

Reply:
I considered having 2 pug versions but felt the the difference was not big enough to warrant having another file in the structure. If there were multiple differences or complicated logic then two would be better but in this case I think one is ok

Reply:
changed to using 2 versions of the view template